test/unit/excluded_channels_test.rb

```diff
@@ -3,15 +3,15 @@
 
 class ExcludedChannelsTest < ActiveSupport::TestCase
   test "excluded_brave_publisher_id? for excluded" do
-    assert Publishers::ExcludedChannels.excluded_brave_publisher_id?("google.com")
+    assert Publishers::ExcludedChannels.excluded_brave_publisher_id?("amazon.com")
```

Comment:
Al can we change this test to not depend on the list at all? e.g. a setup step that runs before any of these tests and replaces the list with one for testing